silvanadomnica_Task3/client.cpp

```diff
@@ -0,0 +1,50 @@
+#include <boost/interprocess/ipc/message_queue.hpp>
+#include <boost/interprocess/managed_shared_memory.hpp>
+#include <boost/interprocess/containers/string.hpp>
+#include <iostream>
+#include <boost/asio.hpp>
+#include "common.h"
+#include <iostream>
+#include <chrono>
+#include <thread>
+
+using namespace boost::interprocess;
+
+int main()
+{
+
+  MessageQueueRequest mqStruct;
+  message_queue mq(open_only, MESSAGE_QUEUE_NAME.c_str());
+
+  typedef boost::interprocess::allocator<char, boost::interprocess::managed_shared_memory::segment_manager> CharAllocator;
+  typedef boost::interprocess::basic_string<char, std::char_traits<char>, CharAllocator> stringShared;
+  managed_shared_memory managed_sm(open_only, SHARED_MEMORY_NAME.c_str());
+  try
+  {
+    while (strcmp(mqStruct.command, "exit") != 0)
+    {
+      cin >> mqStruct.command;
+      mq.send(&mqStruct, MAX_MESSAGE_SIZE, 0);
+      std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
You should add a switch in order to know what response to expect: from add, delete, etc.